ansible/group_vars/tag_env_integration/genesis_accounts.yml

```diff
@@ -0,0 +1,2 @@
+genesis_accounts:
```

Comment:
Is this file name meaningful? I saw changed from accounts.yml in epoch repo, though it looks like name does not matter so not reference to it needs update.

Reply:
No, the filename does not matter from execution point of view. I renamed it to have more meaningful name in this repo context/scope.